src/loader/bgw_launcher.c

```diff
@@ -245,9 +245,14 @@ register_entrypoint_for_db(Oid db_id, VirtualTransactionId vxid, BackgroundWorke
 static HTAB *
 populate_database_htab(void)
 {
+	HASH_SEQ_STATUS hash_seq;
+	DbHashEntry *current_entry;
 	Relation	rel;
 	HeapScanDesc scan;
 	HeapTuple	tup;
+
+	/* Used to store OIDs that can be assigned schedulers */
+	HTAB	   *staging_db_htab = NULL;
```

Comment:
To confirm; this is only being used as a HashSet?

Reply:
Yes. Doc added.